### PR TITLE
Support gazebo harmonic

### PR DESCRIPTION
**Pull request by Ryanf55**
* It's the next release, and is LTS

### CODE REVIEW

File: dev/source/docs/sitl-with-gazebo.rst [hunk 1]
```diff
@@ -45,10 +45,12 @@ Install Gazebo
 
 We currently support
 `Gazebo Garden <https://gazebosim.org/docs/garden/install>`__
-which is available for Ubuntu 20.04 (Focal) and 22.04 (Jammy) and macOS Big Sur
+and
+`Gazebo Harmonic <https://gazebosim.org/docs/harmonic/install>`__
+which are available for Ubuntu 20.04 (Focal) and 22.04 (Jammy) and macOS Big Sur
```
Comment (srmainwaring) [inline dev/source/docs/sitl-with-gazebo.rst:50]: Big Sur (11), Monterey (12), and Ventura (13)

File: dev/source/docs/ros2-gazebo.rst [hunk 2]
```diff
@@ -18,11 +18,11 @@ We will clone the required repositories using `vcstool <https://github.com/dirk-
     wget https://raw.githubusercontent.com/ArduPilot/ardupilot_gz/main/ros2_gz.repos
     vcs import --recursive src < ros2_gz.repos
 
-Set the gazebo version
+Set the gazebo version to either ``garden`` or ``harmonic``
 
 .. code-block:: bash
 
-    export GZ_VERSION=garden
+    export GZ_VERSION=harmonic
```
Comment (srmainwaring) [inline dev/source/docs/ros2-gazebo.rst:25]: I'm confident to recommend using `harmonic` as default in macOS, may need additional testing on Ubuntu before switching?

**Review by srmainwaring — APPROVED** [review]
Thanks @Ryanf55. Couple of optional suggestions.

**Comment by Hwurzburg** [comment]
@Ryanf55  if you are going to make more changes, let me know or I can merge now

File: dev/source/docs/ros2-gazebo.rst (hunk 2 above)
Comment (Ryanf55) [reply dev/source/docs/ros2-gazebo.rst:25]: Yea, I'll leave at garden default for now.

File: dev/source/docs/sitl-with-gazebo.rst (hunk 1 above)
Comment (Ryanf55) [reply dev/source/docs/sitl-with-gazebo.rst]: Done

**Comment by Ryanf55** [comment]
> @Ryanf55 if you are going to make more changes, let me know or I can merge now

Ready for merge once it passes CI.

**Comment by Hwurzburg** [comment]
CI is minimalistic....built it....its fine....merging...

**Pull request merged**